packages/data/src/onboarding/reducer.js

```diff
@@ -3,13 +3,29 @@
  */
```

Comment:
Was there a specific reason this stayed as a JS file?

Reply:
I didn't explore how easy it would be to type this. I'll look into it.

Reply:
I looked at this, the reducer needs some refactoring. It does this cheeky thing where it destructures all the args regardless of what action is taking place, so there is some stuff there that has not been typed yet. I'll come back to improve this later when I do more typing of this package.